_i18n/en.yml

```diff
@@ -599,24 +605,46 @@ policies:
-      Once you reach login.gov your new email address is established.
+    how-do-i-change-my-email-address: To change the email associated with your login.gov account, go to [https://www.login.gov/](https://www.login.gov/) and click “Manage Account” to sign into your login.gov account. Select “edit” next to the email address. Enter the new email address and submit it. Then check your email account and look for an email from login.gov. Open the email and click on the link provided. Clicking the link will redirect you back to login.gov. Once you reach login.gov, your new email address is established.
     how-do-i-change-my-password: |-
-      To change your password, sign in at login.gov and go to your Profile page. If you want to change your password, select Edit next to password, enter the new password and submit your change. Passwords must be at least 8 characters but otherwise there are no restrictions. You can even use more than one word with spaces to get to 8 characters.
+      To change your password, go to https://www.login.gov/ and click “Manage Account” to sign into your login.gov account. Select “Edit” next to password, enter the new password and submit your change. Passwords must be at least 9 characters but otherwise there are no restrictions. You can even use more than one word with spaces to get to 9 characters. Instead, try making a long and complex password by using a phrase or a series of words that only you recognize. And please make it different than any other passwords you use to sign into other accounts such as your bank account or email. Using the same passwords for your email, financial and social media accounts makes identity theft easier.
+
+      [Read more on how to reset your password](site.baseurl/help/signing-in/how-do-i-reset-my-passwordhow-do-i-reset-my-password/)
+
+    how-do-i-change-the-phone-number-i-use-to-sign-in: |-
+      To change the phone number where you get your security code, go to [https://www.login.gov/](https://www.login.gov/) and click “Manage Account” to sign into your login.gov account. Once signed in, select “edit” next to the phone number. Enter the phone number and click “confirm change.” login.gov will immediately send your security code to confirm the new phone number.
+
+      If you cannot receive the security code because you no longer have access to your phone, you will need to [sign in using your personal key](site.basurl/how-do-i-sign-in-if-i-dont-have-my-phone-or-my-phone-number-has-changed). 
+
+    how-do-I-get-a-new-personal-key: |-
+      To request a new personal key, go to [https://www.login.gov/](https://www.login.gov/) and click “Manage Account” to sign into sign into your login.gov account page. Next to personal key click “Get a new key.” Follow the instructions to write down and/or print out the new personal key and click to continue. Remember, you can only request a new personal key if you are able to sign into your login.gov account. 
```

Comment:
In general, we shouldn't be asking folks to click things. Select, choose, follow are all good verbs. Click implies a mouse which is not a safe assumption.

Reply:
@clantosswett ☝️

Reply:
Let's go with select